bpf/polprog/pol_prog_builder.go

```diff
@@ -100,6 +100,7 @@ var (
 	stateOffPostNATDstPort int16 = stateEventHdrSize + 30
 	stateOffIPProto        int16 = stateEventHdrSize + 32
 	stateOffFlags          int16 = stateEventHdrSize + 33
+	stateOffIPSize         int16 = stateEventHdrSize + 34
```

Comment:
Looks like you'll have to remove this one because it's unused, and we have a static check for that.

Reply:
Done.